x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result.tsx

```diff
@@ -75,53 +84,57 @@ export const Result: React.FC<Props> = ({
             showScore={!!showScore}
             isMetaEngine={isMetaEngine}
           />
-          <div className="appSearchResult__body">
-            {resultFields
-              .slice(0, isOpen ? resultFields.length : RESULT_CUTOFF)
-              .map(([field, value]: [string, FieldValue]) => (
-                <ResultField
-                  key={field}
-                  field={field}
-                  raw={value.raw}
-                  snippet={value.snippet}
-                  type={typeForField(field)}
-                />
-              ))}
-          </div>
-          {numResults > RESULT_CUTOFF && !isOpen && (
-            <footer className="appSearchResult__hiddenFieldsIndicator">
-              {i18n.translate('xpack.enterpriseSearch.appSearch.result.numberOfAdditionalFields', {
-                defaultMessage: '{numberOfAdditionalFields} more fields',
-                values: {
-                  numberOfAdditionalFields: numResults - RESULT_CUTOFF,
-                },
-              })}
-            </footer>
-          )}
+          {resultFields
+            .slice(0, isOpen ? resultFields.length : RESULT_CUTOFF)
+            .map(([field, value]: [string, FieldValue]) => (
+              <ResultField
+                key={field}
+                field={field}
+                raw={value.raw}
+                snippet={value.snippet}
+                type={typeForField(field)}
+              />
+            ))}
```

Comment:
This didn't actually change, I just removed a wrapper. I recommend viewing the diff with whitespace changes off (`?w=0`)

Reply:
I think `?w=1` hides whitespace changes, right?

Reply:
oh derp haha, you're right 🤦‍♀️